tasks.py

```diff
@@ -185,9 +185,9 @@ def poll(self, source):
     source.updates['last_activities_cache_json'] = json.dumps(
       {k: v for k, v in cache.items() if k.split()[-1] in silo_activity_ids})
 
-    # Make sure we only fetch the author's h-feed(s) the first time
-    # discover is called
-    is_first_discover = True
+    # Cache to make sure we only fetch the author's h-feed(s) the
+    # first time we see it
+    fetched = set()
```

Comment:
very minor nit: maybe consider `fetched_hfeeds`, since `poll()` is kind of a monstrosity with a ton of other variables and identifiers?  (i mean, come on, who wrote this thing anyway, amirite? :P)